src/exometer_report.erl

```diff
@@ -697,11 +697,11 @@ start_interval_timer(#interval{name = Name, delay = Delay,
     end.
 
 do_start_interval_timer(#interval{name = Name, time = Time} = I, R) ->
-    TRef = erlang:send_after(Time, self(), batch_timer_msg(R, Name)),
+    TRef = erlang:send_after(Time, self(), batch_timer_msg(R, Name, Time)),
     I#interval{t_ref = TRef}.
 
-batch_timer_msg(R, Name) ->
-    {report_batch, R, Name}.
+batch_timer_msg(R, Name, Time) ->
+    {report_batch, R, Name, Time, os:timestamp()}.
```

Comment:
The same thing here. Shouldn't we pass `T0` from the calling function `restart_batch_timer/3` and put it in the message instead of `os:timestamp/0`?